elementor/elementor
# Fix: Fatal error when updating a page with Nested Tabs widget [ED-14534]

## PR Checklist



## PR Type
What kind of change does this PR introduce?


## Summary

This PR can be summarized in the following changelog entry:

* Fixed a fatal error that occurs when an undefined array index is accessed in nested-tabs.php.

## Description
An explanation of what is done in this PR

* In the Nested Tabs widget, there is an issue where the code tries to call print_element() on a potentially non-existent element of the $children array without prior validation. This leads to a fatal error if the $children array at the specified index does not contain an element. The proposed change adds a check to ensure the element exists before attempting to call print_element() on it.

## Test instructions
This PR can be tested by following these steps:

1. Navigate to a page that uses the Nested Tabs widget in Elementor.
2. Ensure that the widget is set up in a way that might previously have triggered the error (e.g., by setting conditions that could lead to an unset index).
3. Verify that the page loads without PHP errors.


## Quality assurance


Fixes #
